CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ## Unversioned
+### Fixed
+- \#2755 - Memory leak in Marathon UI
```

Comment:
Yuppii!!! I guess you can strikethrough the "Partially" from the PR description ;)